store/file/ods_q4.go

```diff
@@ -69,32 +70,35 @@ func OpenODSQ4(pathODS, pathQ4 string) (*ODSQ4, error) {
 	}
 
 	return &ODSQ4{
-		ods: ods,
-		q4Open: func() (*Q4, error) {
-			return OpenQ4(pathQ4)
-		},
+		ods:    ods,
+		pathQ4: pathQ4,
 	}, nil
 }
 
-func (odsq4 *ODSQ4) getQ4() (eds.Accessor, error) {
+func (odsq4 *ODSQ4) tryLoadQ4() {
 	if odsq4.q4Opened.Load() {
-		return odsq4.q4, nil
+		return
 	}
 
 	odsq4.q4Mu.Lock()
 	defer odsq4.q4Mu.Unlock()
 	if odsq4.q4Opened.Load() {
-		return odsq4.q4, nil
+		return
+	}
+
+	q4, err := OpenQ4(odsq4.pathQ4)
+	if errors.Is(err, os.ErrNotExist) {
+		return
 	}
 
-	q4, err := odsq4.q4Open()
 	if err != nil {
-		return nil, fmt.Errorf("failed to open Q4: %w", err)
+		log.Errorf("opening Q4 file %s: %s", odsq4.pathQ4, err)
```

Comment:
shouldn't we handle those?

Reply:
If we fail to open Q4, we still allow reader to read from Q1